test/Driver/sanitize_coverage.swift

```diff
@@ -1,3 +1,5 @@
+// UNSUPPORTED: OS=windows-msvc
```

Comment:
Is this due to the missing functionality in the driver?

Reply:
Yes, I think that was the case back in November. I had found a similar case when trying to build the toolchain with sanitizers enabled here: https://github.com/weliveindetail/swift/commit/5551c86333b511eee23d0c92a2c7798f36761a70

Reply:
Can you file an issue to track re-enabling the set of tests that have been disabled here? The `UNSUPPORTED: OS=windows-msvc` means that we cannot identify when the test starts to work.

Reply:
Right, let's see if they can be XFAIL as well